docs/introduction.md

```diff
@@ -70,14 +70,18 @@ To improve the performance of applications that run in containers, try setting t
 OpenJDK uses the in-built Java cryptographic implementation by default. However, native cryptographic implementations
 typically provide better performance. OpenSSL is a native open source cryptographic toolkit for Transport Layer Security (TLS) and
 Secure Sockets Layer (SSL) protocols, which is well established and used with many enterprise applications. The OpenSSL V1.1.x implementation is
-currently supported for the <!--Digest,--> CBC, GCM, and RSA algorithms.
+currently supported for the ![Start of content that applies only to Java 12](cr/java12.png)Digest![End of content that applies only to Java 12](cr/java_close.png), CBC, GCM, and RSA algorithms.
 
-OpenSSL support is enabled by default for all these supported algorithms. If you want to limit support to specific algorithms, a number of
+![Start of content that applies only to Java 12](cr/java12.png)The OpenSSL V1.0.2 implementation is also supported for the Digest, CBC, GCM, and
+RSA algorithms. On Linux and AIX platforms, the OpenSSL 1.0.2 or 1.1.X library is expected to be found on the system path. (LIBPATH for Linux;
```

Comment:
It doesn't need to be on the LIBPATH, etc. On Linux when you install the openssl package the ldconfig cache is updated to include the libcrypto library. I assume AIX has some similar mechanism.

Reply:
The input I had from Nasser was: 

>The crypto library is not bundled by default. User can set the PATH or LIBPATH or LD_LIBRARY_PATH to crypto library if the library is not already in the system path.

This implies that there might be some situations where you do need to update those env vars. I've modified the text, but if you think I should remove these mentions altogether, I can do.

Reply:
While you can modify the LD_LIBRARY_PATH to include it, it really should be installed via the OS package manager, which uses the ldconfig. Please recommend using the OS package manager first. Not sure we should be recommending manually adding it, but note Linux is LD_LIBRARY_PATH and AIX is LIBPATH.

Reply:
I've changed the wording, removing the setting of any env vars if we are not sure that should be recommended. I guess we have to wait to see where things get to on AIX before merging.

Reply:
AIX is "working", although the changes to make it work haven't been merged yet.

Reply:
The AIX changes are merged now.

Reply:
Do you think this is ready to merge now @pshipton? If so, please go ahead. Latest work staged here: https://eclipse.github.io/openj9-docs-staging/221/introduction/#cryptographic-operations

Reply:
One other issue to resolve.